oeps/processes/oep-0054-core-contributors.rst

```diff
@@ -0,0 +1,249 @@
+
+   #. For candidates coming from an organization, at least one affirmative vote
+      must come from a CC outside of the candidate's org. This is to ensure that
+      core contributors are evaluated on their ability to communicate with the
+      community, outside of their org.
+
+   #. For roles with few existing members, it is encouraged to ask CCs of other
+      roles to help evaluate the candidate.
+
+#. If the candidate is approved, the candidate should proactively work with the
+   Program Administrator to get the access they need for their role. If concerns
+   have been brought up, the sponsor (or Program Administrator, in the case of
+   organic candidates) should discuss with the candidate what they need to work
+   on to become a CC. We encourage the sponsor/Administrator to help the
+   candidate develop a plan of action so that they can be successful in the
+   future.
+   
+Where Do I Start?
+^^^^^^^^^^^^^^^^^
+
+New community members interested in the Core Contributor program should `get in touch with us! <https://open.edx.org/community/connect/>`_
+First check out the `discussion forums <https://discuss.openedx.org/>`_; there are a lot of different
+categories, and we encourage new members to spend some time in the forums, reading through to understand
+what's going on and jumping in to ask questions. There are many people in the `Open edX Slack <https://openedx.slack.com/>`_
+(for those joining for the first time, `here's an invite <http://openedx-slack-invite.herokuapp.com/>`_).
+The ``#core-contributors`` room in Slack can help guide people towards others working in their
+area(s) of interest.
+
+Existing community members who have a record of contributing to the Open edX project should feel free
+to reach out to `current CCs <https://openedx.atlassian.net/wiki/spaces/COMM/pages/3156344833/Current+Open+edX+Core+Contributors>`_
+who have the role they're interested in. Engaging in conversation to see what it's like to be that type of CC
+is invaluable. CCs might even be able to take a look at work the community member has done, however, CCs may
+be pretty overloaded, so offense shouldn't be taken if someone doesn't have time at that moment to help.
+
+If you don't know where to begin, try `joining a working group <https://openedx.atlassian.net/wiki/spaces/COMM/pages/46793351/Working+groups>`_ -
+working groups may have tasks you can pick up to start showing off your skills. Start participating on
+the `discussion forums <https://discuss.openedx.org/>`_; some working groups have an active presence there,
+and you'll get a chance to hone your Open edX expertise by answering questions. And finally, you can find
+core contributors and ask questions about the program and your interests directly either in the
+`Core Contributors discussion category <https://discuss.openedx.org/c/working-groups/core-contributors/36>`_
+or in the ``#core-contributors`` room on Slack.
+
```

Comment:
Could we consolidate all the Program Administrator contact details down here with the rest of the contact links, and reference it where needed?

Putting this at the bottom of the page also guides people to the public channels as a first point of call, instead of sending private emails.

```suggestion
.. _contact_admin:

Contact the Program Administrators
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

Questions about the Program can be directed to the tCRIL administrator(s) via the ``#core-contributors`` Slack room in the `Open edX Slack <https://openedx.slack.com/>`_ or at ``cc-program-admins@tcril.org``.
```

Reply:
This is a good suggestion, very much appreciated